pango/Gir.toml

```diff
@@ -272,9 +271,25 @@ concurrency = "send+sync"
     name = "merge_static"
     ignore = true
 
+[[object]]
+name = "Pango.FontFamily"
+status = "generate"
+    [[object.property]]
```

Comment:
I find it very weird how a ListModel like properties were added but that gir didn't add a `implements GListModel` to glib-wrapper call

Reply:
You need to add `Gio.ListModel` to the `manual` list for that to work